app/src/ui/history/commit-summary.tsx

```diff
@@ -10,20 +10,25 @@ import { IGitHubUser } from '../../lib/dispatcher'
 import { Repository } from '../../models/repository'
 import { CommitIdentity } from '../../models/commit-identity'
 import { Avatar } from '../lib/avatar'
+import { showContextualMenu, IMenuItem } from '../main-process-proxy'
+import { Dispatcher } from '../../lib/dispatcher'
 import { getDotComAPIEndpoint } from '../../lib/api'
+import { Commit } from '../../models/commit'
 
 interface ICommitSummaryProps {
+  readonly dispatcher: Dispatcher
   readonly repository: Repository
   readonly summary: string
   readonly body: string
-  readonly sha: string
+  readonly commit: Commit
```

Comment:
Since we're passing the whole commit in here now, we don't need the separate `summary`, `body`, `author` props.